testing/android_background_image/android/app/build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.application'
 
 android {
+    namespace 'dev.flutter.android_background_image'
```

Comment:
This is the new way of setting this attribute (formerly it was defined in the manifest)